src/UBC Toboggan/Assets/Scripts/beerCollectible.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class beerCollectible : MonoBehaviour
+{
+
+    public float amount = 1f;
+    public float speedMultiplier = 1f;
+    public float magnitudeMultiplier = 0.25f;
+    bool collected = false;
+    float timer = 0f;
+    float yOffset = 0f;
```

Comment:
I don't think this needs to be a field. You can move it's declaration to line 23 and have: 
```cs
float yOffset = (Mathf.Sin(timer*amount*speedMultiplier)+1)*magnitudeMultiplier;
```